akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2StreamHandling.scala

```diff
@@ -256,7 +257,9 @@ private[http2] trait Http2StreamHandling { self: GraphStageLogic with LogHelper
           dispatchSubstream(frame, data, correlationAttributes)
           nextState
 
-        case x => receivedUnexpectedFrame(x)
+        case x =>
```

Comment:
Would be good to have tests for each of these added cases, especially since it seems to apply both to client and server and also to incoming and outgoing streams.

Reply:
Hmm, yeah - I'll roll this back for now and add it again in one or more separate PRs with specific tests.